### PR TITLE
Remove jquery usage

### PR DESCRIPTION
**Pull request by dcyriller**

### CODE REVIEW

File: addon/components/hello-sign.js [hunk 1]
```diff
@@ -136,22 +134,20 @@ export default Component.extend({
...
-          case HelloSign.EVENT_ERROR:
-            self.sendAction('onEventError', eventData);
-            break;
-          default:
-            self.sendAction('onEventInvalid');
-        }
+    options.messageListener = (eventData) => {
+      switch(eventData.event) {
+        case HelloSign.EVENT_SIGNED:
+          this.sendAction('onEventSigned', eventData); // eslint-disable-line
```
Comment (kiwiupover) [inline addon/components/hello-sign.js]: Will this be a breaking change to move these to closure actions?

**Review by kiwiupover — COMMENTED** [review]
This is great!

File: addon/components/hello-sign.js (hunk 1 above)
Comment (dcyriller) [reply addon/components/hello-sign.js]: No, it won't. I just made the update to remove the `sendAction`s.

**Comment by dcyriller** [comment]
Next step: I will add lazy loading of HS script with ember-auto-import.

**Pull request merged**